Softeq.XToolkit.Common.iOS/TextFilters/GroupFilter.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Linq;
+using Foundation;
+using UIKit;
+
+namespace Softeq.XToolkit.Common.iOS.TextFilters
+{
+    /// <summary>
+    ///     This filter combines multiple filters.
+    ///     Returns <see langword="true" /> only if all filters return <see langword="true" />.
+    /// </summary>
+    public class GroupFilter : ITextFilter
```

Comment:
Another option to do that is to create an additional extension method like `public static void SetFilter(this UITextField textField, params ITextFilter filter)` and apply current logic there. I'm not sure if this is a better idea than the current one, but this will allow us to get rid of this filter. @wcoder , what do you think?

Reply:
In this case, we should add the same method for `UITextView` or other future extensions like `UISearchBar`. The current implementation can be reused for any controls in one extension per view.